net/keepalived/files/usr/bin/keepalived-rsync-inotify.sh

```diff
@@ -0,0 +1,14 @@
+#/bin/sh
```

Comment:
Please remove `.sh` from the file name `keepalived-rsync-inotify` that is not common in  `bin` directory

Reply:
removed.